jspsych-uil-utils.js

```diff
@@ -177,6 +179,21 @@ var uil = {};
             // If we do, use that key
             access_key = _access_key;
         }
+
+        let is_online = isOnline(getProtocol(), getHostname());
+
+        if (access_key === KNOWN_INVALID_ID) {
```

Comment:
Maybe also add a check to validate it's formatted like an UUIDv4? Should be doable with a regex. 

Might help if someone accidentally selects all but the last/first character in the UUID

Reply:
Checks added to see whether the access_key has the format of a uuid. It won't differentiate between a uuid v1,2,3,4 or 5, but just the format, I think that is good enough for our purposes.

Reply:
Yeah that's fine; I though of it as an 'oops copy paste error' protection anyway